moped-editor/src/queries/notes.js

```diff
@@ -1,7 +1,7 @@
 import { gql } from "@apollo/client";
 
-export const COMMENTS_QUERY = gql`
-  query GetProjectComments($projectNoteConditions: moped_proj_notes_bool_exp!) {
+export const NOTES_QUERY = gql`
+  query GetProjectNotes($projectNoteConditions: moped_proj_notes_bool_exp!) {
```

Comment:
thanks for updating all these—it's nice lil boost for dev experience.

Reply:
+1 - really nice to have consistent naming in the UI and code 🔍